app/src/main/java/com/sun/chat_04/data/model/User.kt

```diff
@@ -1,13 +1,13 @@
 package com.sun.chat_04.data.model
 
 data class User(
-    val idUser: String = "",
-    val userName: String? = null,
-    val birthday: String? = null,
-    val gender: String? = null,
-    val bio: String? = null,
-    val pathAvatar: String? = null,
-    val pathBackground: String? = null,
+    var idUser: String = "",
```

Comment:
```suggestion
    val idUser: String = "",
```
sửa luôn các trường khác

Reply:
Nhưng mà em muốn update các thông tin của user thì làm sao để val được ạ